AMDirT/core/__init__.py

```diff
@@ -101,6 +101,32 @@ def get_filename(path_string: str, orientation: str) -> Tuple[str, str]:
     elif orientation == "rev":
         return rev
 
+def parse_to_mag(selected_libraries):
+
+    selected_libraries["short_reads_1"] = selected_libraries["download_links"].apply(
+        get_filename, orientation="fwd"
+    )
+    selected_libraries["short_reads_2"] = selected_libraries["download_links"].apply(
+        get_filename, orientation="rev"
+    )
+    selected_libraries["short_reads_2"] = selected_libraries["short_reads_2"].replace(
+        "NA", ""
+    )
+    selected_libraries["longs_reads"] = ""
+    col2keep = [
+        "archive_data_accession",
+        "sample_name",
+        "short_reads_1",
+        "short_reads_2",
+        "longs_reads",
+    ]
+    selected_libraries = selected_libraries[col2keep].rename(
+        columns={
+            "archive_data_accession": "sample",
+            "sample_name": "group",
```

Comment:
Hmm, this is a good question... would a user want to assembly at sample or accession (library level), I sort of feel in aDNA we will rarely want to co-assemble, but I'm not sure. @alexhbnr have you got any opinions here?

Reply:
I think it's a hard question. I would suggest to code it in the way that a sample table is prepared for assembly on sample level because we have quite a number of studies in AncientMetagenomeDir that have more than one sequencing run per sample. So I would keep it as @rozwalak has coded it but would use the column `archive_sample_accession` to set the group, but again the column `sample_name`. The former is a unique identifier, while the latter isn't necessarily. 

Additionally we could add a disclaimer to the docs that we chose to prepare the table like this, but if someone would like to assemble their data differently, you should modify the nf-core/mag input table prior to running the pipeline. What I mean is that some researchers uploaded libraries of the same sample that were differently prepared under multiple `archive_sample_accession` codes, but for assembly one would usually pool them into a single group.

Reply:
@rozwalak what do you think ?

Reply:
Thank you for reminding and sorry for the delay. 

I see one limitation of @alexhbnr solution. Sometimes from the single specimen (`sample_name`) more than one `archive_sample_accession` is assigned. Therefore, using `archive_sample_accession` as a group name may not utilize the full potential of co-assembly. However, I agree that `sample_name` might be not a unique identifier and results in potential troubles in the future. 

I agree that the disclaimer in docs will be valuable and I will change grouping by `sample_name` to `archive_sample_accession` .